app/services/personal_key_generator.rb

```diff
@@ -54,11 +54,11 @@ def hashed_code
     ].join(Pii::Encryptor::DELIMITER)
   end
 
-  def raw_recovery_code
-    @raw_recovery_code ||= RandomPhrase.new(num_words: recovery_code_length).to_s
+  def raw_personal_key
+    @raw_personal_key ||= RandomPhrase.new(num_words: personal_key_length).to_s
   end
 
-  def recovery_code_length
-    Figaro.env.recovery_code_length.to_i || length
+  def personal_key_length
+    Figaro.env.personal_key_length.to_i || length
```

Comment:
renaming the env var is a riskier change, how would you feel about a second PR to do that separately? Or maybe adding code that can read both env vars to ensure a smoother transition?